tests/console/yast2_tftp.pm

```diff
@@ -21,8 +21,12 @@ sub run {
     zypper_call("in tftp yast2-tftp-server", timeout => 240);
 
     script_run("yast2 tftp-server; echo yast2-tftp-server-status-\$? > /dev/$serialdev", 0);
-    assert_screen 'yast2_tftp-server_configuration';
-
+    assert_screen([qw(yast2_tftp-server_configuration yast2_still_susefirewall2)], 90);
+    if (match_has_tag 'yast2_still_susefirewall2') {
+        record_soft_failure "bsc#1059569";
+        send_key 'alt-i';
+    }
+    wait_still_screen 10;
```

Comment:
that's a bit much. Why does the screen need to stay still for 10 seconds?

Reply:
also it's look like wait_still_screen is related to line 27 , so you need to insert it inside 'if' clause

Reply:
no, this is okay to remove: 
1. SuSEfirewall is replaced by firewalld, TW will switch to it soon or later.
2. log view - already has a softfaile because, to be honest I don 't see a chance to get fixed by developer.
3. to focus on main functions is now important: tftp can be enabled and it works.

Reply:
@okurz wait_still_screen 10; is als okay, it might be 5 seconds enough, but I can remove 10 anyway.